rspec-core/lib/rspec/core/configuration.rb

```diff
@@ -1765,14 +1766,57 @@ def self.delegate_to_ordering_manager(*methods)
 
       # Set Ruby warnings on or off.
```

Comment:
Note to self: this is outdated, and we need to document that we accept :none, :all or :deprecations_only.